contrib/drivers/netapp/eseries/model.go

```diff
@@ -0,0 +1,74 @@
+package eseries
+
```

Comment:
Can we have a common model for Netapp SAN? Except few, most of the entities or objects may remain same, example pool, volume or aggregates or LUNs etc,,,